RELEASES.md

```diff
@@ -39,44 +39,42 @@ be done from that branch. For example, once we release `v1.0.0`, a branch
 `release/1.0` will be created from that tag. All future patch releases will be
 done against that branch.
 
-### Pre-releases
+### Release Cadence
 
-Pre-releases, such as alphas, betas and release candidates will be conducted
-from their source branch. For major and minor releases, these releases will be
-done from main. For patch releases, these pre-releases should be done within
-the corresponding release branch.
+Major and minor releases are time based with a target cadence of 6 months. The
```

Comment:
```suggestion
Minor releases are time based with a target cadence of 6 months. The
```

I don't think we want to promise major releases here.  We could also say something akin to "the next release *may* be a major release following the same cadence.